crates/rome_cli/src/stats/formatter.rs

```diff
@@ -0,0 +1,48 @@
+use serde::Serialize;
+use std::collections::HashMap;
+
+#[derive(Debug, Default, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct FormatterStats {
+    /// Useful information of the execution
+    summary: Option<FormatterStatSummary>,
+
+    /// The key is the path of the file
+    details: HashMap<String, FormatterStatDetail>,
+}
+
+impl FormatterStats {
+    pub(crate) fn insert_stat_detail(&mut self, stat: String, detail: FormatterStatDetail) {
```

Comment:
```suggestion
    pub(crate) fn insert_file_content(&mut self, stat: String, detail: FormatterStatDetail) {
```

Can we make `stat` a `Path`? 

Can you document the meaning of `stat` and `detail`. I find it difficult what `stat` is standing for (`path`?)

Reply:
We can't use `Path` because there are situations where we have a `String` and not a `Path`